core/DataAccess/LogAggregator.php

```diff
@@ -195,7 +195,8 @@ public function setQueryOriginHint($nameOfOrigiin)
     private function getSegmentTmpTableName()
     {
         $bind = $this->getGeneralQueryBindParams();
-        return self::LOG_TABLE_SEGMENT_TEMPORARY_PREFIX . md5(json_encode($bind) . $this->segment->getString());
+        $tableName = self::LOG_TABLE_SEGMENT_TEMPORARY_PREFIX . md5(json_encode($bind) . $this->segment->getString());
+        return Common::mb_substr($tableName, 0, 64);
```

Comment:
Can we make 64 a constant? We could also shorten `self::LOG_TABLE_SEGMENT_TEMPORARY_PREFIX` since the table names are about 5 characters too long it seems.

Reply:
Pushed something, is that what you mean?

Reply:
Yep, was also wondering if shortening `self::LOG_TABLE_SEGMENT_TEMPORARY_PREFIX` was an option since `wp_foooobarrrrrr_matomo_logtmpsegment07171d02820b83eab14316299645c2e9` was 69 characters.

Reply:
@diosmosis now I understand what you mean. I guess it's not really needed I would say as the next one might again have an even longer prefix etc. Even if the table name is truncated by 16 characters it would still not be an issue since a temporary table only exists for one session and is not visible to any other mysql session... and after an archive the table is being deleted so it's unlikely there will two tables having the same name... we could shorten it though to `logtmpseg` if that helps and still clear what it is about?

Reply:
I guess it's not really needed. Will merge.